pytest/tests/sanity/routing_table_sync.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3.7
```

Comment:
I’d say just `python3`. We don’t wont it to stop working on systems with Python 3.8 or newer. I only have 3.9 and 3.10 on my system for example.

Reply:
@mina86 Tests don't work with python 3.8.
The command below fails:
```
>>>  python3.8 pytest/tests/sanity/routing_table_sync.py
/home/pmnoxx/repos/nearcore.master/pytest
Traceback (most recent call last):
  File "pytest/tests/sanity/routing_table_sync.py", line 25, in <module>
    from cluster import start_cluster
  File "/home/pmnoxx/repos/nearcore.master/pytest/lib/cluster.py", line 8, in <module>
    import rc
  File "/home/pmnoxx/.local/lib/python3.8/site-packages/rc/__init__.py", line 11, in <module>
    from rc.cache import Cache, CacheCluster
  File "/home/pmnoxx/.local/lib/python3.8/site-packages/rc/cache.py", line 4, in <module>
    from itertools import izip
ImportError: cannot import name 'izip' from 'itertools' (unknown location)
```
This works

```
>>>  python3.7 pytest/tests/sanity/routing_table_sync.py
```

I would prefer to use `#!/usr/bin/env python3` once we fix tests.
However, tests are broken with `python `3.8` and upwards.

Reply:
That’s the wrong `rc` module. Try `python3.8 -m pip install -U -r pytest/requirements.txt`.

Reply:
This is a bit confusing but you have the wrong `rc` installed. Looks like you have [rc](https://pypi.org/project/rc/) while we want [python-rc](https://pypi.org/project/python-rc/). 

```
python3.8 -m pip uninstall rc
python3.8 -m pip install -U -r pytest/requirements.txt
```
will solve your issue.

Reply:
@mina86 Thanks, that fixed the issue.